test/NuGet.Core.Tests/NuGet.Packaging.Test/SigningTests/SignedPackageArchiveUtilityTests.cs

```diff
@@ -399,11 +410,24 @@ public async Task RemoveRepositorySignaturesAsync_WithMultipleCountersignatures_
                 Assert.True(wasSomethingRemoved);
                 Assert.InRange(newLastWriteTime, originalLastWriteTime, originalLastWriteTime.Add(TimeSpan.FromMinutes(5)));
 
+                uint offsetOfLocalFileHeaderLastModifiedDateTime;
+                uint offsetOfCentralDirectoryHeaderLastModifiedDateTime;
+                if (RuntimeEnvironmentHelper.IsWindows)
+                {
+                    offsetOfLocalFileHeaderLastModifiedDateTime = 0x1df;
+                    offsetOfCentralDirectoryHeaderLastModifiedDateTime = 0xa7a;
+                }
+                else
+                {
+                    offsetOfLocalFileHeaderLastModifiedDateTime = 0x1d5;
```

Comment:
In linux and mac the certificates are different than the ones created in windows, therefore the offsets are different.